sos/report/plugins/apache.py

```diff
@@ -48,48 +48,42 @@ class RedHatApache(Apache, RedHatPlugin):
     def setup(self):
         super(RedHatApache, self).setup()
 
-        self.add_copy_spec([
-            "/etc/httpd/conf/httpd.conf",
-            "/etc/httpd/conf.d/*.conf",
-            "/etc/httpd/conf.modules.d/*.conf",
-            # JBoss Enterprise Web Server 2.x
-            "/etc/httpd22/conf/httpd.conf",
-            "/etc/httpd22/conf.d/*.conf",
-            # Red Hat JBoss Web Server 3.x
-            "/etc/httpd24/conf/httpd.conf",
-            "/etc/httpd24/conf.d/*.conf",
-            "/etc/httpd24/conf.modules.d/*.conf",
-        ])
+        # httpd versions, including those used for JBoss Web Server
+        vers = ['', '22', '24']
+
+        # Extrapolate all top-level config directories for each version, and
+        # relevant config files within each
+        etcdirs = ["/etc/httpd%s" % ver for ver in vers]
+        confs = [
+            "conf/httpd.conf",
+            "conf.d/*.conf",
+            "conf.modules.d/*.conf"
+        ]
+
+        # Extrapolate top-level logging directories for each version, and the
+        # relevant log files within each
+        logdirs = ["/var/log/httpd%s" % ver for ver in vers]
+        logs = [
+            "access_log",
+            "error_log",
+            "ssl_access_log",
+            "ssl_error_log"
+        ]
 
         self.add_forbidden_path("/etc/httpd/conf/password.conf")
```

Comment:
Not relevant to this patch itself, but does not the plugin collect `/etc/httpd2[2,4]/conf/password.conf` that we should forbid (using the `vers` variable)?

Reply:
Yep - we should totally expand that.